tests/test_task.py

```diff
@@ -295,7 +295,7 @@ def test_run_cable():
 
     # Success case: run CABLE executable in subprocess
     task.run_cable()
-    assert f"{exe_path} {nml_path}" in mock_subprocess.commands
+    assert f"./{exe_path.name} {nml_path.name}" in mock_subprocess.commands
```

Comment:
I'm suddenly wondering if `exe_path` and `nml_path` really need to exist? Could this assert be: `assert f"./{internal.CABLE_EXE} {internal.CABLE_nml} in mock_subprocess.commands` and remove the definition of `exe_path` and `nml_path`.

That's a detail, I'm happy to put things in as is. Just mentioning it as I like things as simple as possible when we think of it.